Add_user.css

```diff
@@ -20,52 +27,62 @@ body{
 
 nav{
 	width:100%;
-	background-color: white;
+	background-color: #0C59CF;
 	overflow:auto;
 	
 }
 
 ul{
 	padding:0;
-	margin:0 0 0 150px;
+	margin:0 0 0 30px;
 	list-style: none;
 }
-
+/* Set up position for the text in nav to the right */
 li{
-	float:left;
+	float:right;
 }
 
 
-
+/* Set up Drop down menu for */
 nav a{
 	display:block;
 	width:100px;
 	padding: 20px 15px;
 	text-decoration:none;
-	font-family:Arial;
-	color:black;
+	font-family: 'Open Sans', sans-serif;
+	font-weight:bold;
+	color:white;
 	text-align: center;
 }
 
 nav a:hover{
-	background:purple;
+	background:black;
```

Comment:
I change the hover background color to black because the hover background color before was just be the place holder to make sure it work. So when I decided the color theme, I apply color black to be the hover background color because it was one of the color themes that I decided to apply and also it can display a great contrast with navy blue and white.